pkg/datapath/linux/node.go

```diff
@@ -1828,7 +1830,9 @@ func (n *linuxNodeHandler) NodeConfigurationChanged(newConfig datapath.LocalNode
 		}
 	}
 
-	n.updateOrRemoveNodeRoutes(prevConfig.AuxiliaryPrefixes, newConfig.AuxiliaryPrefixes, true)
+	if err := n.updateOrRemoveNodeRoutes(prevConfig.AuxiliaryPrefixes, newConfig.AuxiliaryPrefixes, true); err != nil {
+		return err
+	}
```

Comment:
Just to provide a bit more context on the error:

```suggestion
	if err := n.updateOrRemoveNodeRoutes(prevConfig.AuxiliaryPrefixes, newConfig.AuxiliaryPrefixes, true); err != nil {
		return fmt.Errorf("failed to update or remove node routes: %w", err)
	}
```

Reply:
Right! This is what tracing is for... where does this end if you are in a deep callstack?? net effect errors are a mile long ;( 
This is where I struggled with this pr. At times we add context by wrapping, others we reach out for scoped loggers and hence don't propagate...